src/web/App.css

```diff
@@ -2,6 +2,11 @@ h1 {
   margin-top: 0;
 }
 
+select {
+  height: 21px;
+  width: 177px;
+}
```

Comment:
Just so you're aware, this doesn't scale well based on the font size defined in the accessibility options of the browser. That's why I try to avoid using pixel sizes for any of the controls and texts in the app, and use relative sizes instead (`em`, `rem`, `%`, etc). 

I am aware that if I start doing more complex selects, I might want to use a third-party one. But I hate dependencies on small projects that I have little time to maintain. 😓
The issue with third party ones is that most of them are either:
- they're small and to-the-point, but not accessible,
- they're large and popular, so they include 100 features I don't need and weigh as much as the rest of the the entire app.

So I'm waiting for when I really need it to decide, which will probably be when I implement mods filters ("show me items with all res").